corehq/apps/ota/tests/test_heartbeat.py

```diff
@@ -49,7 +49,7 @@ def _auth_headers(self, user):
 
     def _do_request(self, user, device_id, app_id=None, app_version=1, last_sync='',
                     unsent_forms=0, quarantined_forms=0, cc_version='2.39', url=None,
-                    response_code=200):
+                    response_code=200, fcm_token=''):
```

Comment:
its strange to see a string here in the params, should this be `None`?

Reply:
Added default as  `''` only for it to be easier to include in request params.
Will change this to `None`.

Reply:
Ah yes, this is important thing to be wary of for mutable types. Not applicable to string though.